spec/unit/resources/logical_interconnect_group/load_resource_with_properties_spec.rb

```diff
@@ -70,7 +70,7 @@
     expect_any_instance_of(OneviewSDK::API300::Synergy::LogicalInterconnectGroup).to receive(:add_uplink_set)
 
     ## FC UplinkSet
-    allow_any_instance_of(OneviewSDK::FCNetwork).to receive(:retrieve!).and_return(true)
+    allow_any_instance_of(OneviewSDK::API300::Synergy::FCNetwork).to receive(:retrieve!).and_return(true)
     allow_any_instance_of(OneviewSDK::LIGUplinkSet).to receive(:add_network)
```

Comment:
Please fix this Base module too.

Reply:
This change be in this [LIG PR ](https://github.com/HewlettPackard/oneview-chef/pull/293)